chain/chain/Cargo.toml

```diff
@@ -8,21 +8,21 @@ rust-version.workspace = true
 edition.workspace = true
 
 [dependencies]
-ansi_term = "0.12"
-actix = "0.13.0"
-borsh = "0.9"
-chrono = { version = "0.4.4", features = ["serde"] }
-enum-map = "2.1.0"
-itertools = "0.10.0"
-lru = "0.7.2"
-num-rational = "0.3"
-once_cell = "1.5.2"
+actix.workspace = true
+ansi_term.workspace = true
+borsh.workspace = true
+chrono = { workspace = true, features = ["serde"] }
+crossbeam-channel.workspace = true
+enum-map.workspace = true
+itertools.workspace = true
+lru.workspace = true
+num-rational.workspace = true
+once_cell.workspace = true
 rand = "0.7"
```

Comment:
If we decide to move forward with this PR, do we plan on doing subsequent PRs to synchronise versions for the crates that could not be moved in this PR.

Reply:
I’m working on a change to move all rand to 0.8.5.

Reply:
https://github.com/near/nearcore/pull/7705

Reply:
Either works for me.